internal/aws/cwlogs/cwlog_client.go

```diff
@@ -169,6 +191,17 @@ func (client *Client) CreateStream(logGroup, streamName *string) (token string,
 						}
 					}
 				}
+				logGroupArn := "arn:aws:logs:" + client.region + ":" + client.accountID + ":log-group:" + *logGroup
```

Comment:
Why are we adding the tags after creating the log group?  It seems that there has been a lot of code added to get the region and account ID, including service calls, so that they can be used to construct an ARN for the log group so that tags can be added after the log group is created.  If the tags are added in the `CreateLogGroup` invocation then there's no need for this extra code and effort.

Reply:
I was not aware tags could be added here.  I have removed the extra code and now add tags in the `CreateLogGroup` invocation.